src/flint/types/fmpz_mpoly_q.pxd

```diff
@@ -0,0 +1,9 @@
+from flint.flint_base.flint_base cimport flint_rational_function
+from flint.flintlib.fmpz_mpoly_q cimport fmpz_mpoly_q_t
+
+from flint.types.fmpz_mpoly cimport fmpz_mpoly, fmpz_mpoly_ctx
+
+cdef class fmpz_mpoly_q(flint_rational_function):
+    cdef fmpz_mpoly_q_t fraction
+    cdef fmpz_mpoly_ctx ctx
+    cdef bint _init
```

Comment:
I would rather remove this if it is incomplete.

It would be better if python-flint does not have work in progress implementations of things so that downstream code can depend on something working if it exists e.g.:
```
try:
    from flint import fmpz_mpoly_q
except ImportError:
    # not implemented
else:
    # We should be able to assume that fmpz_mpoly_q actually works
    # rather than being a WIP implementation
```

Reply:
I've removed them for now. I'll resurrect them form the git history when I get around to adding them in another PR